pkg/diagnostics/client_test.go

```diff
@@ -0,0 +1,78 @@
+package diagnostics
+
+import (
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
```

Comment:
We usually make distinct import groups, but it doesn't seem to be covered by linting currently.

See https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Linter is disabled for tests? Should I turn it on?

Reply:
It'd be nice to run some basic linting on tests, maybe not the whole checks that we have, eg. the min-complexity might cause issues on some big table-driven tests.

Reply:
Done

Reply:
Esp. not `errcheck`.

Reply:
Ok, `errcheck` isn't enabled, anyway. 👍🏻